mLRS/Common/esp-lib/esp-rgb-led.h

```diff
@@ -0,0 +1,47 @@
+//*******************************************************
+// Copyright (c) MLRS project
+// GPL3
+// https://www.gnu.org/licenses/gpl-3.0.de.html
+//*******************************************************
+// ESP RGB LED
+//********************************************************
+#ifndef ESPLIB_RGB_LED_H
+#define ESPLIB_RGB_LED_H
+
+#include <NeoPixelBus.h>
+
+// Constants for the state manager
+#define LED_OFF 0
+#define LED_RED 1
+#define LED_GREEN 2
+#define LED_BLUE 3
+
+uint8_t ledCurrentColorState = LED_OFF;
```

Comment:
please initialize this in leds_init()

note that we may soft retsart, and not consistently initializing ALL variables in a init finction but in this way can lead to all sorts of inconsistencies